web/themes/custom/usagov/sass/_wizard.scss

```diff
@@ -69,6 +69,13 @@
         margin-left: 2rem;
       }
 
+      /* We have to have this oneoff selector to override the base uswds style.*/
+      .usa-radio__input:focus+[class*=__label]::before {
+        background-color: #0081a1;
+        box-shadow: 0 0 0 2px #1b1b1b, inset 0 0 0 3px #f0f0f0;
```

Comment:
you should use USWDS color tokens where available, instead of hex values. You can reverse search for colors here https://designsystem.digital.gov/design-tokens/color/system-tokens/ 

In this case, the first gray is `color("gray-90")` the second is `color("gray-5")`

Reply:
looks good